db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -166,6 +167,10 @@ abstract class DbCommonsColumnMappers {
     MappedColumnType
       .base[TxoState, String](_.toString, TxoState.fromString(_).get)
   }
+
+  implicit val feeUnitMapper: BaseColumnType[FeeUnit] = {
```

Comment:
So it's important to be careful with these implicits, because that means we will treat _every_ fee unit as a `SatoshisPerVirtualByte()`, which is definitely what we don't want to do. Unfortunately the only way I know how to handle this is is to not use `implicit`, and just pass in the appropriate mapper where needed, similar to what I did in https://github.com/bitcoin-s/bitcoin-s/pull/884

Reply:
I actually don't think it is an issue to store all `FeeUnit`s as `sats/vbyte` but you have to be careful to actually write correctly, currently `SatoshisPerKiloByte(Satoshis(1)).toLong` returns `1`. You probably need to add a `toSatsPerVByte` to `FeeUnit` interface